### ISSUE TITLE
Quest: The Storm King's Vengance 

### ISSUE THREAD

**Issue by antonius123** [dalaranwow/dalaran-wow#5015]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After phasing in for the quest, I was able to do most of it except for Algar the Chosen, the NPC is missing completely from the phase. I have tried the quest 5 times over a period of 4 hours and he was never there. Tyr answered my ticket and asked me to make a bug report saying the NPC is simply missing.

**Expected behaviour**: Tell us what should happen instead
Algar the Chosen should be flying around Reliquary of Pain in Zul'Drak very close around Voltarus but he was simply missing.
http://www.wowhead.com/npc=29872/algar-the-chosen
Here's the NPC and his location.

**Steps to reproduce the problem**:

1. Take the quest and go to Reliquary of Pain, Algar is missing.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

There is currently an issue with some NPCs "climbing" up into the sky - it has been corrected and it will be active with the next maintenance.

Thank you.

#5011

**Comment by antonius123** [comment]
I looked in the sky and didn't see him however. Just make sure it is the same issue :P
